src/index.js

```diff
@@ -141,7 +141,7 @@ async function handleDelete({ config, handler }, change, log) {
   if (!handler) {
     return {
       status: 400,
-      reason: 'Handler not available, parameters missing or target unsuitable',
+      message: 'Handler not available, parameters missing or target unsuitable',
```

Comment:
I create an `Error` in the main function, so for sake of simplicity, I renamed `reason` to `message` everywhere.